src/app/components/popover/index.html

```diff
@@ -1,13 +1,161 @@
+					</p>
+				</div>
+				<div class="two-col-image">
+					<div class="screenshot">
+						
+					</div>
+				</div>
+			</div>
+		</sky-demo-page-content>
+			
+		<sky-demo-page-content sectionHeading="Behavior & states">
+			<h3>Closing</h3>
+				<div class="sky-text-block">Popovers close when the user clicks on or focuses an element outside the popover.</div>
+			
+			<h3>Responsiveness</h3>
+				<div class="two-col-right-image sky-margin-stacked-separate">
+				<div class="two-col-text">
+					<p>
+						If the popover doesn't fit in the viewport in its current position it will automatically try to reposition itself so that it fits. If it does not fit in any position it will convert itself into a modal overlaying the page.
+					</p>
+				</div>
+				<div class="two-col-image">
+					<div class="screenshot">
+						<img src="~/assets/img/guidelines/popover/behavior-responsive.png" />
+					</div>
+				</div>
+			</div>
+		</sky-demo-page-content>
+		
+		<sky-demo-page-content sectionHeading="Content">
+			<div class="two-col-right-image sky-margin-stacked-separate">
+				<div class="two-col-text">
+					<p>
+						Keep the amount of content in a popover small. The content can be simple text or formatted HTML.
+					</p>
+					<p>
+						Don't include too much content, form inputs or complex interactions.
+					</p>
+				</div>
+				<div class="two-col-image">
+					<div class="screenshot screenshot-success sky-margin-stacked-content-section">
+						<img src="~/assets/img/guidelines/popover/content-simple.png" />
+						<div class="caption sky-text-success">You can use small amounts of plain text content.</div>
+					</div>
+					<div class="screenshot screenshot-success sky-margin-stacked-content-section">
+						<img src="~/assets/img/guidelines/popover/content-html.png" />
+						<div class="caption sky-text-success">You can use small amounts of formatted HTML content.</div>
+					</div>
+					<div class="screenshot screenshot-danger">
+						<img class="bordered" src="~/assets/img/guidelines/popover/content-dont-overload.png" />
+						<div class="caption sky-text-danger">Don't include large amounts of content.</div>
+					</div>
+				</div>
+			</div>
+		</sky-demo-page-content>
+		
+		<sky-demo-page-content sectionHeading="Layout">
+			<div class="two-col-right-image sky-margin-stacked-separate">
+				<div class="two-col-text">
+					<p>In most cases, use the default “above” placement for Popovers. This minimizes the chances that the Popover will obscure content that a user hasn’t seen yet or will interact with next.</p>
+					<p>Use other placement options if the trigger for the Popover will be close to the edge of the display. Choose a placement that will obscure as little other content as possible.</p>
+				</div>
+				<div class="two-col-image">
+					<div class="screenshot sky-margin-stacked-content-section">
+						<img src="~/assets/img/guidelines/popover/layout-alternate.png" />
```

Comment:
Should this image have a caption? If I'm interpreting the caption correctly, it seems like this should have a caption along the lines of "Do use a placement option that minimizes the risk of obscuring content." Or something like that.